### PR TITLE
Update the Readme

### PR DESCRIPTION
**Pull request by Dudejoe870**
This PR I feel like is long overdue, this attempts to update the Readme with accurate information!

I will probably also be attempting to update the website as well, a lot of that information is outdated too.

Feedback is appreciated!

EDIT: You can find the Website PR here: https://github.com/Ryujinx/Ryujinx-Website/pull/2

### CODE REVIEW

**Comment by gdkchan** [comment]
You can remove the `RC1` from the .net core version, since the RTM version has already been released (and 2.2 has been released also).

File: README.md [hunk 1]
```diff
@@ -75,26 +71,25 @@ https://openal.org/downloads/OpenAL11CoreSDK.zip
...
+For help, support, suggestions, or if you just want to get in touch with the team; join our [Discord server](https://discord.gg/VkQYXAZ)!
 
-For donation support, please take a look at our Patreon: https://www.patreon.com/ryujinx
+For donation support, please take a look at our [Patreon](https://www.patreon.com/ryujinx).
 
 **Running**
 
-To run this emulator, you need the .NET Core 2.1 (or higher) SDK *and* the OpenAL 11 Core SDK.  
+To run this emulator, you need the .NET Core 2.1 (or higher) SDK.  
 Run `dotnet run -c Release -- path\to\homebrew.nro` inside the Ryujinx solution folder to run homebrew apps.
```
Comment (gdkchan) [inline README.md]: Should be `inside the Ryujinx project folder`.

File: README.md [hunk 2]
```diff
@@ -75,26 +71,25 @@ https://openal.org/downloads/OpenAL11CoreSDK.zip
 
 **Help**
 
-If you have some homebrew that currently doesn't work within the emulator, you can contact us through our Discord with the compiled NRO/NSO (and source code if possible) and then we'll make changes in order to make the requested app / game work.
+If you have some homebrew that currently doesn't work within the emulator, you can contact us through our Discord with the compiled *.NRO / *.NSO (and source code if possible) and then we'll make changes in order to make the requested app / game work.
```
Comment (gdkchan) [inline README.md]: Would be nice to reword `we'll make changes in order to make the requested app / game work.`.

**Comment by gdkchan** [comment]
I think the default button mapping should be all moved to CONFIG.md or somewhere else, they don't belong in the README.

**Comment by Dudejoe870** [comment]
Feedback has been addressed

**Pull request merged**